module/observationSubmissions/helper.js

```diff
@@ -114,14 +114,25 @@ module.exports = class ObservationSubmissionsHelper {
 
                 let observationSubmissionsDocument = await database.models.observationSubmissions.findOne({
                     _id: observationSubmissionId,
-                    status: "completed"
                 }).lean();
 
+               
                 if (!observationSubmissionsDocument) {
-                    throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND+"or"+messageConstants.apiResponses.SUBMISSION_STATUS_NOT_COMPLETE;
+                    throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND;
                 }
 
-                if( observationSubmissionsDocument.referenceFrom === messageConstants.common.PROJECT ) {
+                const solutionDocument = await database.models.solutions.findOne({
```

Comment:
@rakeshSgr - Use solution helper for the query

Reply:
resolved